packages/pf4-component-mapper/src/form-fields/wizard/wizard-step.js

```diff
@@ -4,38 +4,61 @@ import { Title } from '@patternfly/react-core/dist/js/components/Title/Title';
 import PropTypes from 'prop-types';
 import WizardStepButtons from './step-buttons';
 
-export const RenderTitle = ({ title, customTitle }) => customTitle ? customTitle : <Title headingLevel="h1" size="xl">{ title }</Title>;
+export const RenderTitle = ({ title, customTitle }) => customTitle ? customTitle : <Title headingLevel="h1" size="xl">{title}</Title>;
 
 RenderTitle.propTypes = {
   title: PropTypes.node,
   customTitle: PropTypes.node,
 };
 
-const WizardStep = ({
-  title,
-  description,
-  fields,
-  formOptions,
-  showTitles,
-  showTitle,
-  customTitle,
-  ...rest
-}) => {
-  return (
-    <Fragment>
-      <WizardBody hasBodyPadding={ true }>
-        <div className="pf-c-form">
-          { ((showTitles && showTitle !== false) || showTitle) && <RenderTitle title={ title } customTitle={ customTitle } /> }
-          { fields.map(item => formOptions.renderForm([ item ], formOptions)) }
-        </div>
-      </WizardBody>
-      <WizardStepButtons
-        formOptions={ formOptions }
-        { ...rest }
-      />
-    </Fragment>
-  );
-};
+class WizardStep extends React.Component {
+  formRef = React.createRef();
+  componentDidUpdate(prevProps) {
+    // we want to scroll to top of the new step so
+    // the user experience won't suck. For instance,
+    // when the first step contains many fields that you have to scroll down
+    // to fill all the data for the next step. If the next step contains instructions
+    // at the top, the user will miss them because the scrollbar offset will stay at
+    // the same place it was.
+    if(prevProps.stepKey !== this.props.stepKey) {
+      // HACK: I can not pass ref to WizardBody because it is not
+      // wrapped by forwardRef. However, the step body (the one that overflows)
+      // is the grand parent of the form element.
+      const stepBody = this.formRef.current && this.formRef.current.parentNode && this.formRef.current.parentNode.parentNode;
+      if(stepBody && stepBody.scrollTo && typeof stepBody.scrollTo === 'function') {
+        stepBody.scrollTo({top: 0, left: 0, behavior: 'smooth'});
```

Comment:
I mean.... I don't know why on enzyme scrollTo is not a function but I have to run this check to make tests pass.

Reply:
Enzyme does not recognize s roll because the test env runs on Node and it does not have browser functions and prototypes like Element or scroll. These are only browser specific. To fix this you should add a scroll to polyfill to test setup.

Reply:
If you want to know what functions are missing in test env, I would recommend looking at jasmíne and JSDom, because that is what is jest using to simulate DOM in node.